### DISCUSSION TITLE
table.Create / CreateSet to use transformers

### DISCUSSION THREAD

**Discussion by adae-palfinger** [reqnroll/Reqnroll#137] [Ideas]
We currently use an updated version of this https://gist.github.com/gasparnagy/a478e5b7ccb8f557a6dc to be able to have the transformers execute when using the table.Create and table.CreateSet, is there a new way of handling this?

Would be very good to have it built-in

**Comment by gasparnagy** [comment]
@adae-palfinger We have briefly discussed about this at https://github.com/reqnroll/Reqnroll/pull/108 (related discussion topic https://github.com/orgs/reqnroll/discussions/16), but there is not yet a built-in solution for this. 

But I would like to keep this in mind when we redesign the table helpers, so let's keep this discussion topic to discuss the details.
